eva_submission/nextflow/prepare_brokering.nf

```diff
@@ -52,15 +52,15 @@ process compress_vcf {
     mkdir output
     if [[ $vcf_file =~ \\.gz\$ ]]
     then
-        gunzip -c $vcf_file | $params.executable.bgzip -c > output/$vcf_file
+        $params.executable.bcftools norm -f $fasta -O z -o output/$vcf_file $vcf_file
     else
-        $params.executable.bgzip -c $vcf_file > output/${vcf_file}.gz
+        $params.executable.bcftools norm -f $fasta -O z -o output/${vcf_file}.gz $vcf_file output/${vcf_file}.gz
```

Comment:
[About time](https://ebi-eva.slack.com/archives/CKUT0P8M7/p1564497479075200)... Are there any pitfalls to using "bcftools norm" that could compromise the integrity of the VCF file? I am not aware of any but wanted to check if we should take this opportunity and upgrade to newer versions of bcftools to prevent any unintended consequences.